roles/builder/README.md

```diff
@@ -161,12 +173,26 @@ Path to location of ssh public key to inject into the resulting image to allow
 key-based ssh functionality without extra configuration for systems installed
 with the resulting build media.
 
-builder_password or builder_pub_key needs to be defined when using the kickstart file
+`builder_password` or `builder_pub_key` / `builder_pub_key_path` needs to be defined for system authentication.
```

Comment:
This is going to be a breaking change, anyone already using the role will get an unexpected change in behavior. It's worth either creating a compatibility layer here to deal with `builder_pub_key` getting passed a file path or bumping to version to 2.0.0 and making a changelog snippet as a `breaking_change` entry (https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_changelogs.html#porting-guide-entries-from-changelog-fragments)

Reply:
I don't this will break anything. The current version as we have it is used in a similar way. Per our docs right now it shows a path to the ssh key, which is incorrect. If you look at our default we manually do the lookup on the path. 

So if a user does the same default lookup for `builder_pub_key` it resolves to an ssh key which is what it's expecting.